Tests/SwiftFusionTests/Geometry/Pose3Tests.swift

```diff
@@ -156,4 +156,37 @@ final class Pose3Tests: XCTestCase {
       }
     }
   }
+
+  /// Tests group action: translate only
+  func testActionTranslate() {
+    let x = Vector3(1.0, -2.0, 3.0)
+    let pose = Pose3(Rot3(), Vector3(3.0, -4.0, -5.0))
+
+    let expected = Vector3(4.0, -6.0, -2.0)
+    let actual = pose * x
+
+    assertAllKeyPathEqual(actual, expected, accuracy: 1e-9)
+  }
+
+  /// Tests group action: rotate only
+  func testActionRotate() {
+    let x = Vector3(1.0, -2.0, 3.0)
+    let pose = Pose3(Rot3.fromTangent(Vector3(0.1, 0.2, -0.3)), Vector3(0, 0, 0))
+
+    let expected = Vector3(0.871509606555838, -2.5663299211301474, 2.5796165880985145)
```

Comment:
same, although it might be hard to find a "beautiful" rotation in 3D. I'd have one test with a single yaw pitch or roll which establishes trust, and then a "regression test" with these hardcoded numbers. How do I know these hardcoded numbers are correct in the first place? Comment that :-)

Reply:
Done.